packages/twenty-server/src/engine/api/rest/core/handlers/rest-api-create-many.handler.ts

```diff
@@ -45,6 +49,12 @@ export class RestApiCreateManyHandler extends RestApiBaseHandler {
       overriddenRecordsToCreate.push(overriddenBody);
     }
 
+    await this.createdByFromAuthContextService.injectCreatedBy(
```

Comment:
this function seems to have a side effect which is a bad practice:

let's do something like: 
const overriddenRecordsToCreateWithCreatedBy  = this.createdByFromAuthContextService.injectCreatedBy(overriddenRecordsToCreate...)